### PR TITLE
#1558 powered by neufund in eto dashboard must be a link

### PR DESCRIPTION
**Pull request by AlexeyTuboltsev**

### CODE REVIEW

File: app/components/shared/forms/form-field/utils.ts [hunk 1]
```diff
@@ -41,3 +42,11 @@ export const countedCharacters = (val: InputProps["value"] = "", limit: number)
 
   return `${val.length}/${limit}`;
 };
+
+export const isFieldRequired = (validationSchema: any, name: string) => {
+  if (validationSchema) {
```
Comment (drakkein) [inline app/components/shared/forms/form-field/utils.ts]: can we return false by default if schema is not provided?

File: app/components/shared/forms/form-field/utils.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { FormikErrors, FormikTouched } from "formik";
-import { get } from "lodash";
+import { get, isFunction } from "lodash";
 import { InputProps } from "reactstrap";
+import { getFieldSchema, isRequired } from "../../../../utils/yupUtils";
```
Comment (drakkein) [inline app/components/shared/forms/form-field/utils.ts]: let's stick to convention where 3rd party imports are in first group of imports, and all the others grouped below.

File: app/components/shared/forms/form-field/utils.spec.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import { expect } from "chai";
 
-import { isNonValid, isValid } from "./utils";
+import * as Yup from "yup";
```
Comment (drakkein) [inline app/components/shared/forms/form-field/utils.spec.ts]: If we are grouping imports yup should be moved to first group

File: app/components/eto/overview/EtoOverviewStatus/EtoOverviewStatus.module.scss [hunk 4]
```diff
@@ -155,6 +155,12 @@
   .neufund {
     font-size: 16px;
     margin-top: -2px;
+    text-decoration: none;
+
+    &:hover,
+    &:active {
```
Comment (drakkein) [inline app/components/eto/overview/EtoOverviewStatus/EtoOverviewStatus.module.scss]: Maybe we should change color on focus also? @desfero

File: app/components/shared/forms/form-field/utils.ts (hunk 1 above)
Comment (AlexeyTuboltsev) [reply app/components/shared/forms/form-field/utils.ts]: yes, you're right.
But I'll do it in another branch I'm working on, it landed in this one by mistake

**Pull request merged**